README.md

```diff
@@ -1,13 +1,11 @@
 # Firefish Project - Travel app
 
-This app can be used to plan for travels by building an itenerary. Firefish students and React students will team up to practice everything learned from previous classes.
+This app can be used to plan for travels by building an itinerary. Firefish students will practice everything learned
+from previous classes, and go beyond by integrating with a third-party API.
 
-## Google Drive Link
```

Comment:
Not sure if there was supposed to be a Google Drive link, but the link below is not that.

Reply:
Not sure why i wrote Google Drive. Thank you for correcting it